server/src/setup/http/express/v1-api-router.ts

```diff
@@ -0,0 +1,23 @@
+import { Router } from 'express'
+import { Controllers } from '../../application'
+import { APIRouter } from './api-router'
+import { UserRouter } from '../../../modules/users/infra/http/routes/user-router'
+import { ClubRouter } from '../../../modules/clubs/infra/http/routes/club-router'
+import { EventRouter } from '../../../modules/events/infra/http/routes/event-router'
+
+export const setupV1APIRouter = (controllers: Controllers): APIRouter => {
+  const endpointRouters = {
+    clubs: ClubRouter.using(controllers.getAllClubs),
+    events: EventRouter.using(controllers.getAllEvents),
+    users: UserRouter.using(controllers.createUser),
+  }
+
+  const router = Router()
+  router.get('/', (_req, res) => {
+    return res.json({ message: 'Water, water, water. Loo, loo loo.' })
+  })
+  router.use('/clubs', endpointRouters.clubs)
+  router.use('/events', endpointRouters.events)
+  router.use('/users', endpointRouters.users)
+  return router
+}
```

Comment:
Since our API will be `v1` for the forseeable future, is extracting this file worth the cognitive load? Maybe we can move this into `./versioned-api-router.ts`?